src/sentry/static/sentry/app/components/avatar/styles.tsx

```diff
@@ -13,10 +11,13 @@ export const imageStyle = (props: ImageStyleProps) => css`
   top: 0px;
   left: 0px;
   border-radius: ${props.round ? '50%' : '3px'};
```

Comment:
since `border-radius` is moved to the parent can we remove it now?

Reply:
i tried it, but the img avatars stay square and stick out of the container

Reply:
oof that sucks, aight no worries